### ISSUE TITLE
Can not reproduce the paper-reported acc using default training settings.

### ISSUE THREAD

**Issue by txsing** [bibikar/feddst#3]
Dear Author,

We have followed your instructions and kept all of your code intact, using the default settings of samples_per_client=20, SGD lr=0.01, etc.  We then run the provided command `python3 dst.py --dataset cifar10 --sparsity 0.0` from the readme file. However, the trained global model's performance (accuracy) on the cifar10 test data is still approximately 10%.

We would like to inquire if there are any modifications or changes we can make to the default training settings in order to reproduce the accuracy reported in your paper.

Thank you for your time and assistance.

Best regards
